scripts/dxfToSvg.js

```diff
@@ -51,6 +51,22 @@ function dxfToSvg(dxfString)
                     svgSnippet += getLineSvg(vertice1.x, vertice1.y, vertice2.x, vertice2.y);
                 }
                 return svgSnippet;
+            case 'SPLINE':
+                var svgSnippet = '';
+                var controlPoints = dxfObject.vertices.map((value)=>{return [value.x, value.y]});
```

Comment:
I love ES6, but I'd rather like the code to run on older browsers. Would you use a regular function instead of the arrow function?

(Regarding ES6, `value => [value.x, value.y]` would be shorter.)